Customer Demo/Customer Demo/Screens/SubscriberStatus/SubscriberStatusViewController.swift

```diff
@@ -185,6 +185,13 @@ class SubscriberStatusViewController: UIViewController {
         
         mapTypeButton.setImage(image, for: .normal)
     }
+    
+    func showError(_ error: ErrorInformation) {
+        let alertController = UIAlertController(title: "Error", message: error.message, preferredStyle: .alert)
```

Comment:
I don't think that the ViewController needs the whole ErrorInformation object here. It uses only the `error.message` string here, so might as well just pass the String itself as a parameter. It's a minor thing though, feel free to ignore if you don't feel strongly about this.

Reply:
It was intended. `ErrorInformation` is just a generic struct. I don't think it shouldn't be passed to view.